vars/slingOsgiBundleBuild.groovy

```diff
@@ -189,6 +189,8 @@ def defineStage(def globalConfig, def jobConfig, def jdkVersion, def operatingSy
             }
             // calculate coverage with jacoco (for subsequent evaluation by SonarQube)
             additionalMavenParams = "${additionalMavenParams} -Pjacoco-report"
+            // generate javadocs to detect illegal javadoc markup in sources
+            additionalMavenParams = "javadoc:javadoc ${additionalMavenParams}"
```

Comment:
i thought about this as well, but the variable goal is used in other placed to distinguish between different code paths, and i do not wanted to meddle with this as it might lead to unexpected results if more conditions like this are added later.

the current solution mixing in the additional goal in the additionalMavenParams is also not optimal, but should have not sideffects.

maybe the whole script should be refactored a bit to make it easier to inject additional goals without risking the condition checks based on the primary goal.

Reply:
Ok, let's track this separately.